ubuntu-headless-qt/include/04_mount.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+# --------------------------------------------------------------------------#
+# function run_in_chroot use to run script in ubuntu os after mount chroot.
```

Comment:
This description looks like a function description. Can you update it and move the old description to the correct location of the mentioned functions?